src/entities/nestedJoin/types.ts

```diff
@@ -0,0 +1,48 @@
+import { Address, Hex } from '../../types';
+import { Slippage } from '../slippage';
+import { Token } from '../token';
+import { TokenAmount } from '../tokenAmount';
+
+export type NestedJoinInput = {
+    amountsIn: {
+        address: Address;
+        rawAmount: bigint;
+    }[];
+    chainId: number;
```

Comment:
Can make this `ChainId` type.

Reply:
Sure! It would be interesting to create a separate task to refactor this throughout b-sdk because it's being used as number everywhere.